content/en/Platform Deep Dive/Organization/Organization Settings/SAML SSO/_index.md

```diff
@@ -94,7 +94,10 @@ Learn how to configure SAML SSO with Azure Active Directory (Azure AD) as IdP.
 <br>
 To configure SAML SSO with Azure Active Directory (Azure AD):
 
-1. In Azure AD, create an [enterprise non-gallery application](https://learn.microsoft.com/en-us/azure/active-directory/manage-apps/add-application-portal) for Cobalt.
+1. In Azure AD, create an enterprise non-gallery application for Cobalt.
+    - Select **Enterprise applications**, then select **New application**.
```

Comment:
just showing you the UI. I want to make sure folks don't miss the "Create your own application" https://www.loom.com/share/3a0744b8f7894e908b0c8678a49e26f1

Reply:
Thank you, updated. I have also removed other links to Microsoft instructions.